encoding/delta/binary_packed_amd64.s

```diff
@@ -550,6 +550,11 @@ TEXT ·blockSubInt64AVX2(SB), NOSPLIT, $0-16
     VZEROUPPER
     RET
 
+// vpmaxsq is an emulation of the AVX-512 VPMAXSQ instruction with AVX2.
```

Comment:
Could it make sense to have two versions of this function to leverage AVX-512 when possible ? or would this add too much complexity for the gain ?

Reply:
We could, it doesn't always yield better perf tho, this would require further testing for this particular case.